requirements-test.txt

```diff
@@ -1,5 +1,5 @@
 numpy>=1.16
-scikit-learn
+scikit-learn<1.0
```

Comment:
why do we need to pin the version?

Reply:
Otherwise tests fail due to skopt not being up to date with sklearn 1.0